react/MuiCozyTheme/theme.js

```diff
@@ -42,183 +42,220 @@ export const theme = createMuiTheme({
+      '&$selected, &$selected:hover': {
+        backgroundColor: 'var(--zircon)'
+      },
+      '&:hover, &:focus': {
+        backgroundColor: 'var(--paleGrey)'
       },
-      content: {
-        margin: '0.75rem 0',
-        paddingLeft: '0.5rem',
-        paddingRight: '0.5rem',
-        '& > :last-child': {
-          paddingRight: 0
-        },
-        '&$expanded': {
-          margin: '0.75rem 0'
-        }
+      '&:not(:first-child)': {
+        borderTop: '1px solid var(--silver)'
       }
     },
-    MuiExpansionPanelDetails: {
-      root: {
-        padding: 0,
-        borderTop: '0.0625rem solid var(--silver)'
-      }
+    gutters: {
+      paddingLeft: '1rem',
+      paddingRight: '1rem'
     },
-    MuiList: {
-      root: {
-        borderTop: '1px solid var(--silver)',
-        borderBottom: '1px solid var(--silver)'
-      },
-      padding: {
-        paddingTop: 0,
-        paddingBottom: 0
-      }
+    dense: {
+      minHeight: '3.5rem'
     },
-    MuiListItem: {
-      root: {
-        minHeight: '4.25rem',
-        paddingTop: '.5rem',
-        paddingBottom: '.5rem',
-        '&$selected, &$selected:hover': {
-          backgroundColor: 'var(--zircon)'
-        },
-        '&:hover, &:focus': {
-          backgroundColor: 'var(--paleGrey)'
-        },
-        '&:not(:first-child)': {
-          borderTop: '1px solid var(--silver)'
-        }
-      },
-      gutters: {
-        paddingLeft: '1rem',
-        paddingRight: '1rem'
-      },
-      dense: {
-        minHeight: '3.5rem'
-      },
-      secondaryAction: {
-        paddingRight: '.5rem'
-      },
-      button: {
-        '&:hover': {
-          backgroundColor: getCssVariableValue('paleGrey')
-        }
-      }
+    secondaryAction: {
+      paddingRight: '.5rem'
     },
-    MuiListSubheader: {
-      root: {
-        borderTop: '1px solid transparent',
-        borderBottom: '1px solid var(--zircon)',
-        marginBottom: '-1px',
-        padding: 0,
-        backgroundColor: 'var(--zircon)',
-        textIndent: '2rem',
-        fontWeight: 'bold',
-        fontSize: '.75rem',
-        lineHeight: 1.33,
-        color: 'var(--coolGrey)'
-      },
-      gutters: {
-        paddingLeft: 0,
-        paddingRight: 0
-      },
-      sticky: {
-        backgroundColor: 'var(--zircon)'
+    button: {
+      '&:hover': {
+        backgroundColor: getCssVariableValue('paleGrey')
       }
+    }
+  },
+  MuiListSubheader: {
+    root: {
+      borderTop: '1px solid transparent',
+      borderBottom: '1px solid var(--zircon)',
+      marginBottom: '-1px',
+      padding: 0,
+      backgroundColor: 'var(--zircon)',
+      textIndent: '2rem',
+      fontWeight: 'bold',
+      fontSize: '.75rem',
+      lineHeight: 1.33,
+      color: 'var(--coolGrey)'
     },
-    MuiListItemSecondaryAction: {
-      root: {
-        zIndex: 1
+    gutters: {
+      paddingLeft: 0,
+      paddingRight: 0
+    },
+    sticky: {
+      backgroundColor: 'var(--zircon)'
```

Comment:
So now we can just do `var(--zircon)` instead of `getCssVariableValue('zircon')` ?

Reply:
It depends, see https://github.com/cozy/cozy-ui/pull/1239#discussion_r345648843 for more. Generally though it would probably be safer to always use `getCssVariableValue` going into the future.